common/ellipses.js

```diff
@@ -53,6 +53,8 @@
 
                 }, function deleteFailure(response) {
```

Comment:
I think it's more preferable to have just catch and not both reject callback and catch here.